Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -63,8 +137,33 @@ public static int[] RemoveAt(int[] array, int index)
         /// <returns>A new array with element added at a given index. If an array is empty or null, returns new array with number in it.</returns>
         public static int[] InsertFirst(int[] array, int number)
```

Comment:
What does this have in common with InsertAt? :)

Reply:
return IfArrayNull(array, number);